### PR TITLE
feat: add support for init containers.

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
   <groupId>org.csanchez.jenkins.plugins</groupId>
   <artifactId>kubernetes</artifactId>
-  <version>1.1-SNAPSHOT</version>
+  <version>1.1-test-005</version>
```
Comment (marvinthepa) [inline pom.xml]: Will the maven release plugin like this?

Comment (carlossg) [reply pom.xml]: needs to be removed

Comment (iocanel) [reply pom.xml]: That was not intended! I need to get rid of it! Thanks!

File: README.md [hunk 2]
```diff
@@ -99,7 +99,8 @@ Either way it provides access to the following fields:
 * **name** The name of the pod.
 * **namespace** The namespace of the pod.
 * **label** The label of the pod.
-* **container** The container templates that are use to create the containers of the pod *(see below)*.
+* **initContainers** The init container templates that are use to create the init containers of the pod *(see below)*.
```
Comment (carlossg) [inline README.md]: see below, but no other reference to initContainers ?

**Comment by ClementGautier** [comment]
Init container as pod annotation support have been dropped in 1.8. In my opinion it's not a viable solution to use it like that.

**Comment by carlossg** [comment]
Now supported with yaml syntax

**Comment by caryyu** [comment]
@carlossg if the `yaml syntax` is something like the following in `Jenkinsfile`:

```
...

kubernetes { yamlFile 'KubernetesPod.yaml'}

...
```

which every repository should be repeated the common `init containers` in place, I just wonder how I can define a set of global common `init containers` by the auto-injection?

**Pull request closed without merging**